### ISSUE TITLE
Modeling 과정 검토

### ISSUE THREAD

**Issue by pennya** [freshpotatoteam/DocsCare#4]
#### 4. word embedding layer + gru + softmax
- GRU: Gated recurrent units (GRUs) are a gating mechanism in recurrent neural networks
- softmax:
